### PR TITLE
Hide close button focus unless using keyboard

### PR DESCRIPTION
**Pull request by davidpmccormick**
If the keyboard was used to open the modal window, we outline the element that has keyboard focus (the close button). Otherwise we don't.

### CODE REVIEW

File: catalogue/webapp/components/ExpandedImage/ExpandedImage.js [hunk 1]
```diff
@@ -128,8 +129,14 @@ const CloseButton = styled(Space).attrs({
   background: rgba(0, 0, 0, 0.7);
   color: ${props => props.theme.colors.white};
   border: 0;
+  outline: 0;
   z-index: 1;
 
+  &:focus {
```
Comment (jamesgorrie) [inline catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: So it's focussed and we don't show that it is - seems like an anti-pattern?

Comment (davidpmccormick) [reply catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: It has _keyboard_ focus and we don't show that, unless you're using the keyboard. Seems like a reasonable take?

Comment (jamesgorrie) [reply catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: Not 100% sure I see what the issue is that this solves - maybe easier to show in person?

**Comment by jamesgorrie** [comment]
@davidpmccormick shall we merge this?

**Pull request merged**